File: e-arsip-desa/hashpwd.js
const bcrypt = require('bcryptjs');
const fs = require('fs');

const passwords = {
  'admin123': bcrypt.hashSync('admin123', 10),
  'operator123': bcrypt.hashSync('operator123', 10),
  'kepaladesa123': bcrypt.hashSync('kepaladesa123', 10)
};

console.log(JSON.stringify(passwords, null, 2));

File: e-arsip-desa/hashpwd.py
import bcrypt

passwords = {
    'admin123': bcrypt.hashpw(b'admin123', bcrypt.gensalt()).decode('utf-8'),
    'operator123': bcrypt.hashpw(b'operator123', bcrypt.gensalt()).decode('utf-8'),
    'kepaladesa123': bcrypt.hashpw(b'kepaladesa123', bcrypt.gensalt()).decode('utf-8')
}

for k, v in passwords.items():
    print(f"{k}: {v}")
